### ISSUE TITLE
Failure in boilerplate test

### ISSUE THREAD

**Issue by thangngoc89** [MoOx/statinamic#236]
```
statinamic:builder ✓ Static assets: copy static assets completed +0ms
  statinamic:builder ChunkRenderError: No template for dependency: TemplateArgumentDependency
    at Compilation.createChunkAssets (/home/travis/build/MoOx/statinamic/node_modules/webpack/lib/Compilation.js:874:21)
    at Compilation.<anonymous> (/home/travis/build/MoOx/statinamic/node_modules/webpack/lib/Compilation.js:557:8)
    at Compilation.next (/home/travis/build/MoOx/statinamic/node_modules/tapable/lib/Tapable.js:67:11)
    at ExtractTextPlugin.<anonymous> (/home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/extract-text-webpack-plugin/index.js:285:5)
    at /home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/async/lib/async.js:52:16
    at done (/home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/async/lib/async.js:246:17)
    at /home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/async/lib/async.js:44:16
    at /home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/extract-text-webpack-plugin/index.js:269:6
    at /home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/async/lib/async.js:52:16
    at done (/home/travis/build/MoOx/statinamic/test-boilerplate/node_modules/async/lib/async.js:246:17) +7s
/home/travis/build/MoOx/statinamic/lib/builder/webpack.js:29
      throw new Error("webpack build failed with errors");
```

https://github.com/webpack/webpack/issues/1082#issuecomment-189700673

> Error: No template for dependency: TemplateArgumentDependency happens when you have multiple webpack copies installed and used the DedupePlugin from a different copy.

**Comment by MoOx** [comment]
Will push an improvement and fix soon.

**Comment by MoOx** [comment]
Will push an improvement and fix soon.

**Comment by thangngoc89** [comment]
Thanks. Stuck with #219 because of failing tests for no reason.

**Comment by MoOx** [comment]
Closed by a54e5d11329f35735784770e3066666e506e21e5
